spec/schema.json

```diff
@@ -27,7 +27,7 @@
         "tags": [
           "Healthcheck"
         ],
-        "description": "Check the health of the LFP PAY API Service",
+        "description": "Check the health of the PPS PAY API Service",
```

Comment:
Would this be `Penalty Payment Service (PPS) API`?

Reply:
Will update

Reply:
Can you have another look at the description - I didn't want to say Service twice :)
Unfortunate naming situation with our 3 letter acronym - PPS